Healthy/Classes/Extensions/UITextField+Style.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+// MARK: - TextField style
+//
+extension UITextField {
+    enum TextFieldStyle {
+        case primary
+    }
+}
+
+// MARK: - Apply textfield style
+//
+extension UITextField {
+    func applyTextFieldStyle(_ style: TextFieldStyle) {
+        NSLayoutConstraint.activate([
+            heightAnchor.constraint(equalToConstant: Constants.defaultHeight),
+        ])
+        
+        layer.cornerRadius = Constants.defaultCornerRadius
+        layer.borderWidth = Constants.defaultBorderWidth
+        layer.borderColor = Constants.defaultBorderColor.cgColor
+        
+        setLeftPaddingPoints()
+        setRightPaddingPoints()
+        
+        layer.masksToBounds = true
```

Comment:
?

Reply:
To remove the extra edges in text field.

Reply:
This is a workaround we may not need. Updating the textField `borderStyle` to .none would be enough IMHO.